ksqldb-rest-app/src/main/java/io/confluent/ksql/rest/server/resources/streaming/WSQueryEndpoint.java

```diff
@@ -298,7 +301,19 @@ private void handleQuery(final RequestContext info, final Query query,
           );
       final DataSource dataSource = analysis.getFrom().getDataSource();
       final DataSource.DataSourceType dataSourceType = dataSource.getDataSourceType();
-
+      final Map<String, Object> requestProperties = info.request.getRequestProperties();
+      Optional<ConsistencyOffsetVector> consistencyOffsetVector = Optional.empty();
+      if (ConsistencyOffsetVector.isConsistencyVectorEnabled(requestProperties)
```

Comment:
Is there any reason why this check is different that in other endpoints?
```
if (ksqlConfig.getBoolean(KsqlConfig.KSQL_QUERY_PULL_CONSISTENCY_OFFSET_VECTOR_ENABLED)
```

Reply:
My bad, left over after refactoring